### PR TITLE
Add web editor documentation

### PR DESCRIPTION
**Pull request by ricardonunez-io**

### CODE REVIEW

File: web-editor.mdx [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+
+### Diff View
+
+The **"Diff View"** is a way to view the changes made to a specific document before
+committing it to your repository.
+
+This will help you see changes you've made along with formatting changes made by
+the editor.
+
+## Current limitations
```
Comment (mayankshouche) [inline web-editor.mdx:263]: do openapi files work? should we add as a limitation

File: web-editor.mdx [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+    still have to manually edit renamed, added, and deleted files in your `mint.json`
+  </Accordion>
+
+  <Accordion title="Snippet previews">
+    We currently don't show any previews for custom snippets. This is on our roadmap to support
+    as fully editable components.
+  </Accordion>
+
+  <Accordion title="Keyboard navigation">
+    There are currently some bugs around keyboard navigation that we're working resolving.
```
Comment (mayankshouche) [inline web-editor.mdx]:
```suggestion
    There are currently some bugs around keyboard navigation that we're working to resolve.
```

**Review by mayankshouche — APPROVED** [review]
few nits but overall lgtm!

File: web-editor.mdx (hunk 1 above)
Comment (ricardonunez-io) [reply web-editor.mdx:263]: Ahh very true, thank you kind sir.

File: web-editor.mdx (hunk 2 above)
Comment (ricardonunez-io) [reply web-editor.mdx]: Fixed, thank you kindly

**Pull request merged**